source/_components/conversation.markdown

```diff
@@ -20,6 +20,10 @@ To enable the conversion option in your installation, add the following to your
 ```yaml
 # Example configuration.yaml entry
 conversation:
+    boolean_test:
```

Comment:
Avoid changing the default example.
Create an advanced example a sin other components where you document the advanced features.
Documentation for configuration variables are also missing.